client/my-sites/plans/main.jsx

```diff
@@ -411,6 +412,7 @@ class Plans extends Component {
 				{ selectedSite.ID && <QuerySitePurchases siteId={ selectedSite.ID } /> }
 				<DocumentHead title={ translate( 'Plans', { textOnly: true } ) } />
 				<PageViewTracker path="/plans/:site" title="Plans" />
+				<QueryProductsList />
```

Comment:
The products list is necessary to load add-ons data properly, which we now require in the `/plans` page